static/styles/rewards/media-queries.css

```diff
@@ -18,6 +18,26 @@ header span:first-child::after {
   }
 }
 
+@media screen and (max-width: 565px){
+  #rewardRecipient > a {
+    display: inline-block;
+    width: 100%;
+    max-width: calc(0.5*100vw);
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+}
+
+@media screen and (min-width: 565px) {
+  #rewardRecipient > a {
+    display: inline-block;
+    width: 100%;
+    max-width: calc(0.7*565px);
```

Comment:
What is the significance of 0.7 and 565?

Reply:
The table body width is 565px max. On bigger screens we can increase the anchor length as long as it doesn't overflow the table body container.

On small screens however, the green width is usually below 565px therefore the table will assume view width size. The parent element , td#recipient, of the anchor tag is 80% of the table body width. Taking 80% of 80 leaves us with 64% of view width.

Reply:
Why did you make it 70%

Reply:
That's the most optimal value

Reply:
> Taking 80% of 80 leaves us with 64% of view width.

There's nothing about 70% in your explanation